docs/develop.md

```diff
@@ -158,44 +163,60 @@ cd /var/www/securedrop/
-```
-
-In order to access the servers after the install is completed you will need to install and configure a proxy tool to proxy your SSH connection over Tor. Torify and connect-proxy are two tools that can be used to proxy SSH connections over Tor. You can find out the SSH addresses for each server by examining the contents of `app-ssh-aths` and `mon-ssh-aths` in `/install_files/ansible-base`. Also you must add the HidServAuth values to your `/etc/tor/torrc` file and reload Tor.
+In order to access the servers after the install is completed you will need to install
+and configure a proxy tool to proxy your SSH connection over Tor.
+Torify and connect-proxy are two tools that can be used to proxy SSH connections over Tor.
 
 ### connect-proxy (Ubuntu only)
 
 Ubuntu: `sudo apt-get install connect-proxy`
-*Note: you used to be able to install connect-proxy on Mac OS X with Homebrew, but it was not available last we checked (Wed Oct 15 21:15:17 PDT 2014).*
+*Note: you used to be able to install connect-proxy
+on Mac OS X with Homebrew, but it was not available last we checked (Wed Oct 15 21:15:17 PDT 2014).*
 
-After installing connect-proxy via apt-get, you can use something along the lines of the following example to access the server. Again you need Tor running in the background.
+After installing connect-proxy via apt-get and appending the tor config options
+to your local config, you can export the environment variable `SECUREDROP_SSH_OVER_TOR=1`
+in order to use `vagrant ssh` commands to access the prod instances. Here is an example
+of how that works:
 
 ```
-ssh admin1@examplenxu7x5ifm.onion -o ProxyCommand="/usr/bin/connect-proxy -5 -S localhost:9050 %h %p"
+$ vagrant up --provision /prod/     # restricts SSH to Tor after final reboot
+$ vagrant ssh-config app-prod       # will show incorrect info due to lack of env var
+Host app-prod
+  HostName 127.0.0.1
+  User vagrant
+  Port 2201
+  UserKnownHostsFile /dev/null
+  StrictHostKeyChecking no
+  PasswordAuthentication no
+  IdentityFile /home/conor/.vagrant.d/insecure_private_key
+  IdentitiesOnly yes
+  LogLevel FATAL
+$ vagrant ssh app-prod -c 'echo hello'   # will fail due to incorrect ssh-config
+ssh_exchange_identification: read: Connection reset by peer
+$ export SECUREDROP_SSH_OVER_TOR=1       # instruct Vagrant to use Tor for SSH
+$ vagrant ssh-config app-prod            # will show correct info, with ProxyCommand
+Host app-prod
+  HostName l57xhqhltlu323vi.onion
+  User vagrant
+  Port 22
+  UserKnownHostsFile /dev/null
+  StrictHostKeyChecking no
+  PasswordAuthentication no
+  IdentityFile /home/conor/.vagrant.d/insecure_private_key
+  IdentitiesOnly yes
+  LogLevel FATAL
+  ProxyCommand connect -R remote -5 -S 127.0.0.1:9050 %h %p
+$ # ensure ATHS values are active in local Tor config:
+$ cat *ssh* | sudo tee -a /etc/tor/torrc > /dev/null && sudo service tor reload
+$ vagrant ssh app-prod -c 'echo hello'   # works
```

Comment:
Although one still has to update their `torrc` manually to add the `HidServAuth` values.